docs/logging.md

```diff
@@ -1,58 +1,68 @@
 # Logging
 
-Using Bigflow, you can easily add a handler to your root logger that will send all logging calls to the Cloud Logging.
-In the CL, you can filter logs per project id/logger name and workflow id. 
 
-## Getting started
-To use logging, you have to install `bigflow` with `logger` module - `bigflow[log]`.
-To create a logger with the CL handler, you have to use `configure_logging` method.
+With Bigflow you can easily configure Python logging to send all messages the Cloud Logging (Stackdriver).
+Cloud Logging allows you to filter logs by project id / log name / workflow id and other fields.
 
-```python
-from bigflow.log import BigflowLogging
-project_id = 'some_project'
-logger_name = __name__
-
-
-BigflowLogging.configure_logging(project_id, logger_name)
-```
+To use logging, you have to install `bigflow` with `log` module - `bigflow[log]`.
+Then you need to pass logging configuration to your `Workflow`.
 
-or, if you prefer to filter CL logs by workflow id:
- 
 ```python
-from bigflow.log import BigflowLogging
-project_id = 'some_project'
-logger_name = __name__
-workflow_id = 'some-workflow'
+import bigflow
+
+gcp_project_id = 'some_project_id'
+
+workflow = bigflow.Workflow(
+    name='workflow_name',
+    log_config={
+        'project_id': gcp_project_id,
+        'level': 'DEBUG',
+    },
+    # ...
+)
+```
 
+BigFlow will automatically configure `logging` when workflow is executed with `bigflow run` or `bigflow deploy` commands.
+If you want to run workflow manually from your custom script, then you need to call method on `Workflow` object manually:
 
-BigflowLogging.configure_logging(project_id, logger_name, workflow_id)
+```python
+import bigflow
+
+gcp_project_id = 'some_project_id'
+
+workflow = bigflow.Workflow(
+    name='workflow_name',
+    log_config={
+        'project_id': gcp_project_id,
+        'level': 'DEBUG',
+    },
+    # ...
+)
+
+def run_manually():
+    workflow.init_logging()
+    workflow.run("2018-01-01")
+    workflow.run("2018-01-02")
 ```
-After calling the `configure_logging` method, your calls to any logger will be send to CL.
 
-Also in a console, you should see a
-link to the CL query. In the case of the code above, the link should look like this `https://console.cloud.google.com/logs/query;query=logName%3D%22projects%2Fsome-project%2Flogs%some-logger%22%0Alabels.id%3D%some-workflow%22\`
-
-To send logs to the CL, you use the standard python `logging` module.
+In order to send logs into the Cloud Logging you should the standard python `logging` module.
```

Comment:
In order to send logs into the Cloud Logging you should use the standard python `logging` module.

Reply:
done.